README.md

```diff
@@ -1,4 +1,4 @@
-# Vue.js SoundCloud Web Player
+# Vue.js SoundCloud Web Player ([Check it out](https://shahzeb1.github.io/soundcloud-webplayer/))
```

Comment:
It is. Sorry about that. I didn't mean to commit the README.md to this pull request 😭 

Will look into fixing it! LOL